oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCClientConfiguration.java

```diff
@@ -1116,6 +1126,24 @@ public GroupMapping getGroupMapping()
         return groups;
     }
 
+    /**
+     * @since 2.18.3
+     */
+    public String getGroupMappingIncludeRegex()
```

Comment:
Feels cleaner to directly return a Pattern (it would also avoid recompiling this pattern for each group name, which is what you are currently doing by using `String#matches()`).

Reply:
will do.

Reply:
I just pushed a commit in this PR that handles this.